core/contracts/financial-templates/implementation/ExpiringMultiParty.sol

```diff
@@ -5,5 +5,13 @@ import "./Liquidatable.sol";
 
 
 contract ExpiringMultiParty is Liquidatable {
+    /**
+     * @notice Note on party members of this registered financial contract.
```

Comment:
nit: this comment is inconsistent with our standard. the `@notice` should define the contract construction like
```
     * @notice Construct the ExpiringMultiparty
```

nit2: The `@dev` description you have looks good but I think you've exceeded the 120 char limit (the first line is 161 chars). Try wrapping them to all be about the same length ending below 120 chars.

nit3: if you are going to use natspec here then you need to define the `@param`

Reply:
Actually I'm curious where you think we should document the rules on what constitutes the "party" associated with the registered EMP financial-contract.